OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -40,9 +42,3429 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    በዝ፡ ዓለም፡ እምኔነ፡ ኢትሰሰሊ<hi rend="rubric">፨</hi> ወበዘይመጽእ፡ ዓለም፡ ወልድኪ፡ ነፍስነ፡ እምነፍስኪ፡ ኢይሌሊ<hi rend="rubric">፨</hi>
+                                    ለዓለመ፡ ዓለም፡ አሜን<hi rend="rubric">፨</hi>
+                                </explicit>
+                            </msItem>
+                            
+                        
+                        <msItem xml:id="ms_i1.2">
+                            <locus from="1va" to="2rb"/>
+                            <title ref="LIT5443Salamlaki2" type="complete"/>
+                      <textLang mainLang="gez"/>
+                            <incipit>
+                                <locus from="1va" to="1vb"/>
+                                <hi rend="rubric">ሰላም፡ ለ<add place="above">ኪ</add><supplied reason="omitted" resp="DR">፡</supplied></hi> ኦእግዝእትየ፡
+                                <hi rend="rubric">ማርያም፡</hi> ድንግል፡ እመ፡ <cb n="b"/> መድኅን፡ ወእመ፡ ብርሃን<hi rend="rubric">፨</hi>
+                                እመ፡ ቍርባን<hi rend="rubric">፤</hi> ወእመ፡ ዕጣን<hi rend="rubric">፤</hi> እመ፡ ነድ<hi rend="rubric">፤</hi> ወእመ፡
+                                ዋሕድ<hi rend="rubric">፨</hi> እመ፡ ንጉሥ፡ ወእመ፡ ክርስቶስ፡ እመ፡ መሲሕ<hi rend="rubric">፤</hi> ወእመ፡ መለኮታዊ፡ መፍርህ<hi rend="rubric">፨</hi>
+                            </incipit>
+                            <explicit xml:lang="gez">
+                                <locus target="#2rb"/>
+                                ቅዱሳን፡ ወንጹሓን፡ ድንግልናኪ፡ እለ፡ ይሴብሑ<hi rend="rubric">፨</hi> ኀበ፡ ሥእልኪ፡ እለ፡ ይጸርሑ፡ ብርሃነ፡ ገጽኪ፡ ይትሞጥሑ<hi rend="rubric">፨</hi>
+                                ኀበ፡ ቤተ፡ መርዓሁ፡ ለወልድኪ<hi rend="rubric">፤</hi> ያንፈርዕፁ፡ ወይትፌሥሑ<hi rend="rubric">፤</hi> ለዓለመ<hi rend="rubric">፤</hi>
+                                ዓለም<hi rend="rubric">፤</hi> አሜን<hi rend="rubric">፨ ፨ ፨</hi>
+                            </explicit>
+                        </msItem>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i2">
+                            <locus from="7ra" to="19rb"/>
+                            <title>Excerpts from the New Testament</title>
+                            
+                        <msItem xml:id="ms_i2.1">
+                            <locus from="7ra" to="7vb"/>
+                            <title ref="LIT3517Epistle" type="incomplete">2 Corinthians 8:1-15</title>
+                            <textLang mainLang="gez"/>
+                            
+                            <incipit xml:lang="gez">
+                                <locus target="#7ra"/>
+                                <hi rend="rubric">ወእነግረክሙ፡ አኃዊነ፡ ጸጋ፡ እግዚአብሔ</hi>ር፡ ዘተውህበ፡ ለቤተ፡ ክርስቲያን፡ ዘመቂዶንያ<hi rend="rubric">፨</hi>
+                                እስመ፡ እምብዙኀ፡ መከራ፡ ሕማሞሙ፡ ፈድፈደ፡ ፍሥሐሆሙ፡ ምስለ፡ ዕመቀ፡ ንዲቶሙ፡ ፈድፈደት፡ ፍሥሐ፡ ብዕሎሙ<hi rend="rubric">፨</hi>
+                                እስመ፡ <gap reason="illegible" unit="chars" extent="1"/>ልየ፡ ሰማዕቶሙ፡ ከመ፡ በአምጣነ፡ ኃይሎሙ<hi rend="rubric">፨</hi>
```

Comment:
The barely readable letter is ለ, but if we want to indicate that in this condition the letter is mostly illegible, it's ok!